website/content/en/platforms/arm.adoc

```diff
@@ -77,7 +99,7 @@ Listing all supported devices on all CPUs and boards is impractical here, howeve
 * TI OMAP
 * Xilinx Zynq7 family.
 
-Note that not all peripherals are supported on all CPUs.
+Not all peripherals are supported on all CPUs.
```

Comment:
This paragraph seems superfluous, repetitive. 

[`d5ecf98` (#215)](https://github.com/freebsd/freebsd-doc/pull/215/commits/d5ecf982d667f8895b4831acd55cffceeb222a6c#diff-06532acdcf84a18734f78e5ee58d1ee12f58c3353c542a6b57eac4e3fe71615bR56) forewarns that _not every peripheral is supported on every CPU or board_.